bot/cogs/error_handler.py

```diff
@@ -147,10 +148,18 @@ def __init__(self, bot: Bot):
             f"Sorry, an unexpected error occurred. Please let us know!\n\n"
             f"```{e.__class__.__name__}: {e}```"
         )
-        log.error(
-            f"Error executing command invoked by {ctx.message.author}: {ctx.message.content}"
-        )
-        raise e
```

Comment:
Is this still going to be logged to stdout? (by the way, in #757 I changed this to use `exec_info` instead of raising)

Reply:
log.error should go to stdout shouldn't it?

Reply:
Well yes but the point is that you've completely removed these lines.

Reply:
Will use exc_info though, I like that

Reply:
Oh wait, I didn't understand this right, I removed the log.error, adding back.

Reply:
Since you're still doing `capture_exception`, will Sentry not end up with duplicate entries for this exception?

Reply:
Yeah, I've removed the capture_exception and added log.error:

<img width="973" alt="image" src="https://user-images.githubusercontent.com/20439493/75058001-19be7900-54d2-11ea-9134-41c35ffae1a2.png">

<img width="857" alt="image" src="https://user-images.githubusercontent.com/20439493/75058053-35c21a80-54d2-11ea-81c7-b1839ce62f9e.png">

Reply:
It still gets those extra tags set?

Reply:
Yeah, because I run the log command from inside a scope I've created